examples/python2.7/Dockerfile

```diff
@@ -1,8 +1,45 @@
 FROM python:2.7-slim AS build-env
+
+# Using standard Python image
+#  Add in the application code
+#  Byte-compile Python code to *.pyc
+
 ADD . /app
 WORKDIR /app
+RUN python -m compileall .
+
+# Copy the results into the distroless image.
 
 FROM gcr.io/distroless/python2.7
```

Comment:
Is this stage necessary?

Reply:
Yes, this is an example of building the *.pyc to be dropped into the distroless image so they are not byte compiled on the fly each time it starts up.  I wanted a demonstration of dropping a *.py in and then creating an image with the byte compiled *.pyc as a simple fist step before showing off the pip install requirements step.

The comment is a little redundant but I felt like adding more comments.

Reply:
Hmm... I mean you have the same stages below as `FROM python:2.7-slim AS bytecompile` and `FROM gcr.io/distroless/python2.7 AS compiled`, so I thought this top two stages are unnecessary? They look identical. I don't see them in your Python 3 example either.

Reply:
You might be right the github interface was confusing me, one second while I go to the source

Reply:
You were correct, thanks for catching that. corrected and pushed, building now. Thanks!